src/site/content/_redirects.yaml

```diff
@@ -215,3 +215,8 @@ redirects:
 
 - from: /image-support-for-async-clipboard
   to: /async-clipboard
+
+# Vanity URLs
+
+- from: /goibibo
+  to: /goibibo-case-study
```

Comment:
do you think we need a vanity url for this? Are we planning to put something else at `/goibibo` ?

Reply:
I realized it was unlikely we would need `web.dev/goibibo` for anything else so I changed it to that and removed the redirect